.github/workflows/lint.yml

```diff
@@ -36,11 +36,13 @@ jobs:
           go-version: 1.19
       - name: Clone the code
         uses: actions/checkout@v3
-      - name: Run linter
+      - name: Run linter for codebase
         uses: golangci/golangci-lint-action@v3
         with:
           version: v1.50
           only-new-issues: true # Show only new issues if it's a pull request
+      - name: Run linter for scaffolded projects
+        run: make lint-test
```

Comment:
It seems not working because we have no failures and I do not think that the projects that will scaffold would pass successfully in the check.